grammars/css.cson

```diff
@@ -498,7 +498,13 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
-        'match': '(\\.)[a-zA-Z0-9_-]+'
+        'match': '''(?x)
+          (\\.)
+          -?+                   # possessive to avoid matching '-' followed by <0-9>
+          [\\w-&&[^0-9]]        # Do NOT change 0-9 to \d.
```

Comment:
http://regex101.com failing me now with these advanced regexes...

Can you clarify that comment to explicitly say that the two aren't the same?